settei/presets/celery.py

```diff
@@ -179,14 +182,68 @@ def worker_config(self) -> typing.Mapping[str, object]:
         )
         return celery_config
 
-    def on_worker_loaded(self, app):
-        """Be invoked when a Celery app is ready.
+    @typechecked
+    def on_worker_loaded(self, app: Celery):
+        """Trigger the ``worker.on_loaded`` hooks.
+        You should invoke this function when the Celery app is ready
+        with the Celery app as argument.
+        You may want to use
+        :attr:`celery.loaders.base.BaseLoader.on_worker_init`
+
+        ``worker.on_loaded`` hook can be a Python code or list of module path.
+
+        When ``worker.on_loaded`` is a single string, it will be interpreted as
+        Python code.
+        The configuration and the Celery app is injected as
+        ``self`` and ``app`` each:
+
+        .. code-block:: toml
+
+            [worker]
+            on_loaded = \"""
+            print('Hello, world!')
+            print('self is configuration!: {}'.format(self))
+            print('app is celery app!: {}'.format(app))
+            \"""
+
+
+        When ``worker.on_loaded`` is a list of string, it will be interpreted
+        as module paths:
+
+        .. code-block:: toml
+
+            [worker]
+            on_loaded = [
+                "utils.hooks:sample_hook",
+                "src.main:print_hello_world",
+            ]
+
+        The hook must receive two arguments, :class:`Configuration` and
+        :class:`celery.Celery`::
+
+            def sample_hook(conf: Configuration, app: Celery):
```

Comment:
How about using the `code-block`? It is easy to confuse indent when `::` syntanx and `code-block` were mixed. And I think keeping a one style on one context is preferred for code maintaining.